element_array_ephys/plotting/widget.py

```diff
@@ -0,0 +1,90 @@
+import ipywidgets as widgets
+from .. import ephys_report
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+import pathlib
+import typing as T
+from IPython.display import display
+from .. import ephys_no_curation as ephys
+from skimage import io
+
+
+# Widgets
+probe_dropdown_wg = widgets.Dropdown(
+    options=ephys.CuratedClustering & ephys_report.ProbeLevelReport.fetch("KEY"),
```

Comment:
You don't need to fetch the keys to restrict by another subquery.
```suggestion
    options=ephys.CuratedClustering & ephys_report.ProbeLevelReport.proj(),
```